content/04.experiment-types.md

```diff
@@ -16,4 +16,80 @@ Proteomics experiments can be both qualitative and quantitative.
+[@doi:10.1038/s41592-019-0459-y] gives recommendations on how to prep samples, conduct data analysis, and present findings in a detailed stepwise manner that will be useful to have if you are new to using this technique.
+
+3. Fast photochemical oxidation of proteins (FPOP ) [@doi:10.1074/jbc.REV119.006218]
+Fast photochemical oxidation of proteins (FPOP) is a laser based HRPF (hydroxyl radical protein footprinting) MS method that relies on the irreversible labeling of solvent-exposed amino acid side chains by hydroxyl radicals in order to understand structure elucidation of proteins. The laser is set at 248 nm and will cause the hydrogen peroxides to become hydroxyl radicals. [@doi:10.1016/j.jasms.2005.09.008; @doi:10.1016/j.ijms.2006.08.018] The flow rate and laser frequency are adjusted in the flow system so that each protein molecule is radiated only once. After they are radiated, the sample is collected to a tube that has catalase and free methionine in buffer so that the H2O2 and OH can be quench and prevent further unwarranted oxidation. Control samples are made through running the sample through the flow system without any radiation. Another experimental control is done by adding a radical scavenger (glutamine is popularly used) with the purpose of preventing over oxidation. [@doi:10.1002/pro.3408; @doi:10.1021/ac901054w] FPOP has wide application for proteins, to include being able to report fast protein folding and protein transient dynamics.
+
+4. protein-painting [@doi:10.1038/ncomms5413; @doi:10.1016/S0959-440X(02)00283-X]
+Protein painting is an optimized MS-based technique in which first the samples are coated with paint molecules. These paint molecules will coat the surfact but will not have access to the protein-protein interface regions that solvents cannot get to. The parts that are painted will not be subjected to trypsin cleavage while the unpainted areas will. After proteolysis, the peptides samples will be subjected to MS. The samples will only show the unmodified opposing regions in the protein-protein interaction at the time of the painting. 
+
+5. LiP-MS (limited proteolysis mass spectrometry) [@doi:10.1016/j.tibs.2020.05.006]
+LiP-MS is a two-step process that begins with the separation of proteins of interest using a high-performance liquid chromatography (HPLC) system. The separated proteins are then subjected to a liquid-phase micro-sampling (LPMS) procedure, which allows for the analysis of a small number of proteins at a time. This methodology allows for the identification and quantification of the proteins of interest without the need for sample pre-treatment or proteomic analysis.
```

Comment:
```suggestion
LiP-MS is a two-step process that begins with the separation of proteins of interest using a high-performance liquid chromatography (HPLC) system. 
The separated proteins are then subjected to a liquid-phase micro-sampling (LPMS) procedure, which allows for the analysis of a small number of proteins at a time. 
This methodology allows for the identification and quantification of the proteins of interest without the need for sample pre-treatment or proteomic analysis.
```

Reply:
This is not how I would describe the limited LiP-MS procedure. Can you please re write to answer the following: What is the goal? How does it work? I have not heard of LPMS where does LPMS fit in and what is it? How is it possible to identify without proteomic analysis?